eng/versioning/version_shared.py

```diff
@@ -144,10 +144,8 @@ def update_change_log(setup_py_location, version, service, package, is_unrelease
         service,
         "--PackageName",
         package,
-        "--Unreleased",
-        str(is_unreleased),
-        "--ReplaceLatestEntry",
-        str(replace_version)
+        f"--Unreleased:${is_unreleased}",
```

Comment:
I don't think the `:` is needed you should be able to pass these as individual args.

Reply:
Unfortunately passing individual arguments does not work for Booleans. It errors with "argument of type 'bool' is not iterable". If you add the quotes then it fails when converting from string to PowerShell boolean. I think the way I have done it is the best option.

Reply:
I feel like we are still missing something here but I'm also OK with using the `:` if it makes things a little easier.